app/controllers/follows_controller.rb

```diff
@@ -0,0 +1,23 @@
+class FollowsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_question, only: [:create, :destroy]
+
+  authorize_resource
```

Comment:
So? How do you think `authorize_resource` works? How does it know, where you define the variable? `authorize_resource` is almost equal to:

```ruby
class FollowsController < ApplicationController
  ...
  
  before_action :authorize_resource_hook
  ...
  
  private

  def authorize_resource_hook
    # there is not @follow yet, we're in before hook, before the action itself is called 
    resource = instance_variable_get(":@#{controller_name.singularize}")
    # resource is nil here
    authorize! action_name, resource
  end
end
```